### PR TITLE
Top layer update

### PR DESCRIPTION
**Pull request by burakukula**
Before:
screen shot 2016-12-14 at 09 22 36

After:
screen shot 2016-12-14 at 09 22 17

### CODE REVIEW

File: src/components/toplayer/_toplayer.scss [hunk 1]
```diff
@@ -1,6 +1,5 @@
-$toplayerCrossColor: $white;
 $toplayerBackgroundColor: $white;
-$toplayerShadow: $defaultBottomShadow;
+$toplayerShadow: 0 2px 6px 3px $defaultBottomShadowColor;
```
Comment (kdzwinel) [inline src/components/toplayer/_toplayer.scss]: what was wrong with `defaultBottomShadow`?

**Comment by kdzwinel** [comment]
Screenshot please!

File: src/components/toplayer/_toplayer.scss (hunk 1 above)
Comment (burakukula) [inline src/components/toplayer/_toplayer.scss]: @kdzwinel It was requested by Greg, `$defaultBottomShadow` have `0 2px 6px 0`.

**Comment by burakukula** [comment]
@kdzwinel please have a look now. sorry about the screeshots..

**Pull request merged**